ChangeLog.d/getrandom.txt

```diff
@@ -0,0 +1,2 @@
+Features
```

Comment:
I think this should probably be in the "Changes" section instead: it's not really adding new functionality, but providing a better implementation of existing functionality.